opentelemetry-sdk/src/metrics/internal/mod.rs

```diff
@@ -22,10 +23,7 @@ use crate::metrics::AttributeSet;
 pub(crate) static STREAM_OVERFLOW_ATTRIBUTES: Lazy<Vec<KeyValue>> =
     Lazy::new(|| vec![KeyValue::new("otel.metric.overflow", "true")]);
 
-pub(crate) trait Aggregator<T>
-where
-    T: Number,
-{
+pub(crate) trait Aggregator {
```

Comment:
is this change doable as a standalone, separate PR, to keep the PRs shorter and easier to review.?

Reply:
Done, created separate revision for this #2269.
Updated this revision with git rebase, (I think it's ok, as it basically deleted code that shouldn't be there)